crates/bevy_math/src/aspect_ratio.rs

```diff
@@ -11,17 +11,60 @@ use bevy_reflect::Reflect;
 pub struct AspectRatio(f32);
 
 impl AspectRatio {
-    /// Create a new `AspectRatio` from a given `width` and `height`.
+    /// Standard 16:9 aspect ratio
+    pub const SIXTEEN_NINE: Self = Self(16.0 / 9.0);
+    /// Standard 4:3 aspect ratio
+    pub const FOUR_THREE: Self = Self(4.0 / 3.0);
+    /// Standard 21:9 ultrawide aspect ratio
+    pub const ULTRAWIDE: Self = Self(21.0 / 9.0);
+
+    /// Create a new [`AspectRatio`] from a given width and height.
     #[inline]
     pub fn new(width: f32, height: f32) -> Self {
         Self(width / height)
     }
 
-    /// Create a new `AspectRatio` from a given amount of `x` pixels and `y` pixels.
+    /// Create a new [`AspectRatio`] from a given amount of x pixels and y pixels.
     #[inline]
     pub fn from_pixels(x: u32, y: u32) -> Self {
         Self::new(x as f32, y as f32)
     }
+
+    /// Create a new [`AspectRatio`] from a given width and height.
+    /// Returns None if height is zero or if the result is not finite.
+    #[inline]
+    pub fn try_new(width: f32, height: f32) -> Option<Self> {
```

Comment:
Just joining in on this, but I think a `try_` method in this particular case should return a `Result`, because if we are going to be checking to make sure this works well, we also should guard against `NaN` values. And because of that, we have multiple different failure modes:

1. Infinites widths/heights
2. Zero widths/heights
3. NaN widths/heights

Being able to determine *what* is breaking would be really useful.